src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -58,6 +67,31 @@ export const RegistrationSearchBar = ({ aggregations }: RegistrationSearchBarPro
       </Field>
       <RegistrationSortSelector />
 
+      <LoadingButton
+        variant="outlined"
+        startIcon={<FileDownloadIcon />}
+        loadingPosition="start"
+        sx={{ gridArea: 'export' }}
+        loading={isLoadingExport}
+        onClick={async () => {
+          setIsLoadingExport(true);
+          try {
+            const fetchExportData = await fetchRegistrationsExport(window.location.search);
+            const blob = new Blob([fetchExportData], { type: 'text/csv' });
+            const url = URL.createObjectURL(blob);
+            const link = document.createElement('a');
+            link.download = 'result-export.csv';
+            link.href = url;
+            link.click();
```

Comment:
mener mye av dette skulle vært mulig å få til med en vanilla `<a>`-tag, men får ikke det til å virke helt som ønsket, da den åpner i ny fane i stedet for å laste ned som fil 🤔 
```
<a href="https://api.dev.nva.aws.unit.no/search/resources" download="file.csv">
  Eksporter
</a>
```